controller.py

```diff
@@ -93,6 +93,8 @@ def __init__(self, app, local_path='', progress_bar=None):
             'language': {
                 'cs_CZ': 'Czech',
                 'en_US': 'English'
+				'de_DE': 'Deutsch'
```

Comment:
This is using tabs instead of whitespaces, also it's missing semicolons and gives syntax errors